### ISSUE TITLE
Build fails when a "code-block" has the option emphasize-lines and the number indicated is higher than the number of lines

### ISSUE THREAD

**Issue by carlossc** [sphinx-doc/sphinx#3770]
[test.zip](https://github.com/sphinx-doc/sphinx/files/1013037/test.zip)
### Problem
The process of building the output of a Sphinx project breaks when a `code-block` directive has the option `:emphasize-lines:` and one of the numbers indicated is higher than the number of lines in the code-block

#### Procedure to reproduce the problem

1. Download and uncompress the attached zip file [test.zip](https://github.com/sphinx-doc/sphinx/files/1013037/test.zip)
2. Go into the project and run make html

#### Error logs / results
```
reading sources... [100%] My Test
Exception occurred:
  File "C:\Python35-32\lib\site-packages\sphinx\directives\code.py", line 130, in run
    (nlines, self.options['emphasize_lines']),
KeyError: 'emphasize_lines'
```
#### Expected results
Instead of failing, Sphinx should print the warning and continue the built.

> line number spec is out of range(1..."

### Environment info
- OS: Windows and Linux
- Python version: Python 2.7.5 and Python 3.5.1
- Sphinx version: 1.6.1 (the latest at the time of opening the bug)

**Comment by carlossc** [comment]
This bug is minor. However, as I am not experienced at all with Python, it took me a while to figure out that this was the problem. I thought it was due to upgrading to the latest version of Sphinx.

I already have the solution. I am going to create a pull request for this.

**Comment by tk0miya** [comment]
Thank you for reporting.
Fixed at a43cac5.
